test_generators/shuffling/main.py

```diff
@@ -0,0 +1,54 @@
+from eth2spec.phase0 import spec
+from eth_utils import (
+    to_dict, to_tuple
+)
+from gen_base import gen_runner, gen_suite, gen_typing
+from preset_loader import loader
+
+
+@to_dict
+def shuffling_case(seed: spec.Bytes32, count: int):
+    yield 'seed', '0x' + seed.hex()
+    yield 'count', count
+    yield 'shuffled', [spec.get_permuted_index(i, count, seed) for i in range(count)]
+
+
+@to_tuple
+def shuffling_test_cases():
+    for seed in [spec.hash(spec.int_to_bytes4(seed_init_value)) for seed_init_value in range(30)]:
+        for count in [0, 1, 2, 3, 5, 10, 33, 100, 1000]:
+            yield shuffling_case(seed, count)
+
+
+def mini_shuffling_suite(configs_path: str) -> gen_typing.TestSuiteOutput:
```

Comment:
Same suggestion as deposit tests. When generator is config agnostic, recommend just using the same function that takes `config` as a param

Reply:
See deposit suite comment